pyccel/ast/core.py

```diff
@@ -2132,6 +2143,16 @@ def rename(self, newname):
 
         self._name = newname
 
+    def add_local_var(self, var):
+        """
+        Add a new variable to the local variables tuple
+
+        Parameters
+        ----------
+        var : Variable
+              The new local variable
+        """
+        self._local_vars = self._local_vars + (var,)
```

Comment:
Is it not easier to use a list to store the local variables?

Reply:
All `_attribute_nodes` are tuples. I could increase all the tests to include any iterable if you prefer? I don't think this is costly though as we only use it when creating additional variables in the printing stage which doesn't happen very often

Reply:
Don't worry, it's not important.